dev/build_release.py

```diff
@@ -434,6 +454,51 @@ def start_copy_file(self, source, target):
         shutil.copy(source, target)
         return NO_PROCESS
 
+  def start_copy_debian_target(self, name):
```

Comment:
Does this use the 'simple' debian index or the structured one when it copies to the remote debian repository?